src/core/thread/context.d

```diff
@@ -28,6 +28,10 @@ struct StackContext
     StackContext* within, next, prev;
 }
 
+
+/// Flag to control rethrow behavior of call, join and friends
+enum Rethrow : bool { no, yes }
```

Comment:
I don't know how good of an idea is to put it here. But basically the reason that it can't go inside `StackContextExecutor` is that it's needed by `ThreadGroup`.

Reply:
Thats fine, this is also user facing code so I think it needs to anyway.